includes/class-wp-job-manager-post-types.php

```diff
@@ -651,6 +662,71 @@ public function job_feed() {
 		remove_filter( 'posts_search', 'get_job_listings_keyword_search' );
 	}
 
+	/**
+	 * Modifies WordPress Query of public search.
+	 *
+	 * @param WP_Query $query Query being processed.
+	 */
+	public function maybe_hide_filled_expired_job_listings_from_search( Object $query ): void {
```

Comment:
This `Object` parameter type is a little bit odd in here it should be `WP_Query` **but** we [recently discovered](https://senseip2.wordpress.com/2023/02/21/sensei-pro-1-11-0-post-mortem/) that typing the parameters for hook callbacks might give problems.

So better removing the type completely.